macGist/GitHubCredential.swift

```diff
@@ -8,21 +8,48 @@
 
 import Foundation
 
-enum GitHubCredential: String {
-    case clientSecret
-    case clientID
+enum GitHubCredentialManager {
+    static private let keychain = Keychain()
     
-    var value: String {
-        guard let path = Bundle.main.path(forResource: "Credentials", ofType: "plist"),
-            let dict = NSDictionary(contentsOfFile: path) as? [String: AnyObject] else { fatalError("No plist found") }
+    static var clientId: String? {
+        get {
+            do {
+                let clientId = try keychain.readGitHubClientId()
+                return clientId
+            } catch {
```

Comment:
if you aren't really using this catch, will not be better if you use:
```swift
return try? keychain.ReadGitHubClientId()
```
?

Reply:
Suggestion accepted.